apps/web/src/app/api/documents/[projectId]/for-import/route.ts

```diff
@@ -0,0 +1,34 @@
+import { Workspace } from '@latitude-data/core/browser'
+import { BadRequestError } from '@latitude-data/core/lib/errors'
+import { DocumentVersionsRepository } from '@latitude-data/core/repositories'
+import { authHandler } from '$/middlewares/authHandler'
+import { errorHandler } from '$/middlewares/errorHandler'
+import { NextRequest, NextResponse } from 'next/server'
+
+export const GET = errorHandler(
+  authHandler(
+    async (
+      _: NextRequest,
+      {
+        params,
+        workspace,
+      }: {
+        params: { projectId: string }
+        workspace: Workspace
+      },
+    ) => {
+      const { projectId } = params
+
+      if (!projectId) {
+        throw new BadRequestError('Project ID is required')
+      }
```

Comment:
Can we use zod for input validation in API routes like we do with server actions?

Reply:
we would need to create a middleware for that, totally doable